applications/nrf_desktop/doc/usb_state.rst

```diff
@@ -205,3 +209,22 @@ The |usb_state| is a transport for :ref:`nrf_desktop_config_channel` when the ch
 
 The module also handles a HID keyboard LED output report received through USB from the connected host.
 The module sends the report using :c:struct:`hid_report_event`, that is handled either by :ref:`nrf_desktop_hid_state` (for peripheral) or by the :ref:`nrf_desktop_hid_forward` (for dongle).
+
+nRF54H20 support
+================
+
+Due to the nRF54H20 USB device controller (UDC) characteristics, following changes have been made in the USB state module to support the nRF54H20 platform:
+
+* The USB state module spawns a separate thread to handle initializing, enabling, and disabling the USB stack.
+* The module disables the USB stack when the USB cable is disconnected and enables the stack when the cable is connected.
+
+Those changes are only applied to the nRF54H20 platform and do not affect other platforms.
+They are necessary to ensure proper USB stack operation on the nRF54H20 platform.
```

Comment:
```suggestion
These changes are applicable to the nRF54H20 platform only.
They are necessary to ensure proper USB stack operation on the nRF54H20 platform.
```

Though, technically this whole chapter is named "nRF54H20 support", and this fact is repeated at the beginning of it, so I'd say it's pretty self-explanatory - maybe this part isn't needed? 

```suggestion
These changes are necessary to ensure proper USB stack operation on the nRF54H20 platform.
```

Reply:
I'd leave it. The changes that we made in USB state module that were necessary for nrf54h20 could have possibly affect other platforms that also use USB state module. Here, we explicitly say that they haven't.